ansibleinviewer/tests/test_inventoryparser.py

```diff
@@ -0,0 +1,69 @@
+import os
+from parameterized import parameterized
+import unittest
+import yaml
+
+from ansibleinviewer.inventoryparser import InventoryParser
+
+
+class TestInventoryParser(unittest.TestCase):
+
+    all_hosts = ['10.0.0.4', '10.0.0.5',
+                 '172.16.0.30', '172.16.0.43', '172.16.0.8',
+                 '192.168.0.2', '192.168.0.3']
+    groupA_hosts = ['10.0.0.5', '172.16.0.30', '192.168.0.2']
+    groupB_hosts = all_hosts
+    groupC_hosts = ['172.16.0.43']
+    groupD_hosts = ['10.0.0.4', '172.16.0.8', '192.168.0.3']
+    groupE_hosts = ['10.0.0.4', '172.16.0.8', '192.168.0.3']
+    groupF_hosts = all_hosts
+    groupAC_hosts = groupA_hosts+groupC_hosts
+    groupDC_hosts = groupD_hosts+groupC_hosts
+
+    @staticmethod
+    def load_inventory_file(inventory_path: str):
+        with open(inventory_path) as inventory_file:
+            inventory_data = yaml.safe_load(inventory_file)
+        return inventory_data
+
+    def setUp(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        inventory_data = self.load_inventory_file(f'{ dir_path}/files/inventory.yml')
```

Comment:
The inventory.yml file was not pushed as I've added `inventory.yml` to the .gitignore. You can remove it from the .gitignore or rename the test file.